api/src/config/config.rs

```diff
@@ -106,6 +106,7 @@ impl Config {
         })?)
     }
 
+    // TODO: no result
```

Comment:
did you forget about this todo?

Reply:
Yes, totally forgot about it. Now it is fixed.